### ISSUE TITLE
Using ZeroTextureData in Firefox 64+ spams this warning

### ISSUE THREAD

**Issue by crazyquark** [mrdoob/three.js#15419]
Using the latest Firefox 64(or Firefox Developer edition) you get this warning all the time:
`Error: WebGL warning: clear: This operation requires zeroing texture data. This is slow.`
You can reproduce this in a lot of the three.js official examples like:
https://threejs.org/examples/#webgl_animation_cloth for example.

There is a reference in the Gecko code about this here:
https://github.com/mozilla/gecko-dev/blob/1cdb198bd9e1d2605ac48d2588e29475fb01a74e/dom/canvas/WebGLTexture.cpp#L636-L647

**Comment by Mugen87** [comment]
Yes, I can reproduce the warning with FF 64.0 and Mac OS 10.14.1. I think it makes sense to create a small [fiddle](https://jsfiddle.net/f2Lommf5/) that produces such a warning.

**Comment by crazyquark** [comment]
OK, I will try to edit it.

**Comment by Mugen87** [comment]
Wait, I already have a fiddle^^:

https://jsfiddle.net/f2Lommf5/16472/

Looks like it happens as soon as a shadow map is rendered.

**Comment by crazyquark** [comment]
Yes, I can see the warning on your fiddle as well.
Disabling the shadowMap removes the warning.

**Comment by Mugen87** [comment]
I've debugged the issue and I think the problem is right here:
https://github.com/mrdoob/three.js/blob/3013895bf68ad0c27fb0f9d3ca0beea2c0e7a87a/src/renderers/webgl/WebGLShadowMap.js#L226

The problem is that the shadow maps render target is cleared in the same method invocation where it is created. You can simulate the same with the following fiddle:

https://jsfiddle.net/f2Lommf5/16482/

No problem in Chrome but FF logs the respective warning.

**Comment by Mugen87** [comment]
According to other posts, we might just ignore the warning:

https://github.com/pixijs/pixi.js/issues/4496#issuecomment-369510971

http://www.html5gamedevs.com/topic/39057-this-operation-requires-zeroing-texture-data-this-is-slow/?tab=comments#comment-223038

**Comment by RemusMar** [comment]
You won't get this warning in the upcoming Firefox 65.

**Comment by Mugen87** [comment]
Indeed, with Firefox Beta (65.0b4) I'm not able to reproduce the warning on macOS.
